docs/customization/fixtures.rst

```diff
@@ -97,100 +101,42 @@ Adding new items in shop by standard yaml configuration:
                                         - "HUN_WEB"
                                     enabled: true
 
-Fixtures in sylius are loaded and initialized using class located at ``src/Sylius/Bundle/CoreBundle/Fixture/*``
-
-Below is presented class ``Sylius\Bundle\CoreBundle\Fixture\CurrencyFixture`` that load our currencies from yaml configuration:
-
-.. code-block:: php
-
-    <?php
-
-    ...
-
-    class CurrencyFixture extends AbstractFixture
-    {
-        ...
+Products are a more complicated fixture, because they have more dependencies (to Variants, Options etc.). In order to prepare a Product
```

Comment:
```suggestion
It is more complicated to create fixtures for products because they have more dependencies (to variants, options etc.). In order to prepare a product
```